frontend/src/pages/DetailsPage.tsx

```diff
@@ -1,6 +1,7 @@
 import {Workout} from "../types/Workout.ts";
 import {useNavigate, useParams} from "react-router-dom";
 import axios from "axios";
+import "/src/App.css"
```

Comment:
i think we should do the styling in a css-file for each component. in app.css there should only be stylings that adress everything - and i dont think, we need to import it then.